tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -169,7 +169,7 @@ def _receive_data_on_session_stream(self, data: bytes, fin: bool) -> None:
                 code = buffer.pull_uint32()
                 # TODO(yutakahirano): Make sure `reason` is a
                 # UTF-8 text.
-                reason = buffer.data
+                reason = buffer.pull_bytes(len(capsule.data) - 4)
```

Comment:
Can we have a comment why we subtract 4?

Also is it always guaranteed that `len(capsule.data) >= 4`?

Reply:
Done.

> Also is it always guaranteed that `len(capsule.data) >= 4`?
Yes, otherwise the above `pull_uint32()` throws.

Reply:
I see, thanks!